ipfx/stimulus_protocol_analysis.py

```diff
@@ -100,6 +100,9 @@ def analyze(self, sweep_set):
 
     def as_dict(self, features, extra_params=None):
         out = features.copy()
+        out.pop("sweeps")
+        out.pop("spikes_set")
```

Comment:
Since it doesn't look like you are doing anything with the popped values from the dictionary, I think it's more explicit if you instead do:
```
del out["sweeps"]
del out["spikes_set"]
```